packages/core/src/llm/providers/local/node-llama-provider.ts

```
@@ -0,0 +1,309 @@
+
+import type { GPUInfo } from './types.js';
+import { LocalModelError } from './errors.js';
+import { detectGPU } from './gpu-detector.js';
+
+/**
+ * Check if node-llama-cpp is installed.
+ */
+export async function isNodeLlamaCppInstalled(): Promise<boolean> {
+    try {
+        // Try to require the module at runtime
+        await eval('import("node-llama-cpp")');
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)